index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
```

Comment:
- Kindly add a css file to style your index.html file which will enable you to add styling to your markup.
- Also remember to link it in the ```<head>``` section of your index.html. 
example:
```<link rel="stylesheet" href="styles.css">```